addons/batikvis/src/main/java/de/lmu/ifi/dbs/elki/visualization/svg/SVGPath.java

```diff
@@ -651,7 +653,7 @@ private void append(String action, double... ds) {
    * @return path object, for compact syntax.
    */
   public SVGPath close() {
-    if(lastaction != SVGConstants.PATH_CLOSE) {
+    if(!Objects.equals(lastaction, SVGConstants.PATH_CLOSE)) {
```

Comment:
Here, too, the actions are constants that can be safely compared with identity; which is more efficient.